nni/algorithms/compression/v2/pytorch/pruning/tools/task_generator.py

```diff
@@ -42,13 +43,16 @@ def __init__(self, total_iteration: int, origin_model: Module, origin_config_lis
             The log directory use to saving the task generator log.
         keep_intermediate_result
             If keeping the intermediate result, including intermediate model and masks during each iteration.
+        skip_zero_iteration
+            If skipping the first iteration, the iteration counter will start at 1.
```

Comment:
then the name should `skip_first_iteration`

Reply:
also better to explain when should we set this argument to be true

Reply:
fixed and add docstr
```
        skip_first_iteration
            If skipping the first iteration, the iteration counter will start at 1.
            In these function-based iterative pruning algorithms, iteration `0` means a warm up stage with `sparsity = 0`.
            If the `original_model` is a pre-trained model, the first iteration is usually can be skipped.
```